docs/docs/introduction.md

```diff
@@ -0,0 +1,48 @@
+# Introduction
+
+Redwood is the full-stack web framework designed to help you grow from side project to startup.
+Redwood features an end-to-end development workflow that weaves together the best parts of [React](https://reactjs.org/), [GraphQL](https://graphql.org/), [Prisma](https://www.prisma.io/), [TypeScript](https://www.typescriptlang.org/), [Jest](https://jestjs.io/), and [Storybook](https://storybook.js.org/).
+
+## Getting the Most out of Redwood
+
+To get the most out of Redwood, do two things:
+
+- Start the [Tutorial](tutorial/foreword.md)
+- Join the [community](https://community.redwoodjs.com/)
```

Comment:
```suggestion
- Join the [community](https://redwoodjs.com/community)
```

FYI: I'm going to create a page/post tomorrow that will be our Go-to for community onboarding. I'll make sure the redirect goes to the page I create (if not that post)